src/main/java/com/minecolonies/coremod/event/FMLEventHandler.java

```diff
@@ -80,7 +80,7 @@ public void onPlayerLogin(@NotNull final PlayerEvent.PlayerLoggedInEvent event)
         {
             MineColonies.getNetwork().sendTo(new ServerUUIDMessage(), (EntityPlayerMP) event.player);
             MineColonies.getNetwork().sendTo(new ColonyStylesMessage(), (EntityPlayerMP) event.player);
-            ColonyManager.syncAllColoniesAchievements();
+            //ColonyManager.syncAllColoniesAchievements();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)